package.json

```diff
@@ -36,7 +35,8 @@
     "test:ci-sandbox": "npm run test:ci-any --project=sandbox",
     "test:ci-shared": "npm run test:ci-any --project=shared",
     "test:ci": "npm run test:ci-sandbox && npm run test:ci-shared && npm run test:ci-metis",
-    "test:e2e-any": "npm run tsc-test-data --project=${npm_config_project} && ((node --experimental-specifier-resolution=node --use_strict ./projects/${npm_config_project}/dist/out-tsc/projects/${npm_config_project}/test-data/index.js) & (npm run cypress:ci-any --project=${npm_config_project}; fuser -k 3000/tcp))",
+    "prep-test-data": "node tools/test-data-prep/rename-files-to-mjs.mjs projects/${npm_config_project}/dist/out-tsc",
```

Comment:
I think we should add some pretty detailed documentation to describe what we're doing here and why.

Reply:
Done (added comment above script)